lib/redis.js

```diff
@@ -29,7 +29,10 @@ module.exports = class Redis {
     // optionally run in cluster mode
     if (redisUrl.match(/^cluster:\/\/.+:[0-9]+$/)) {
       const parts = redisUrl.replace(/^cluster:\/\//, '').split(':')
-      this.conn = new IORedis.Cluster([{host: parts[0], port: parts[1]}], {redisOptions: opts})
+      this.conn = new IORedis.Cluster([{host: parts[0], port: parts[1]}], {
+        clusterRetryStrategy: opts.retryStrategy,
```

Comment:
ok, I see how this `opt.retryStrategy` was defined above, and now passed in to the constructor as a peer to `redisOptions` - makes sense.